index.html

```diff
@@ -3325,89 +3325,47 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
     JSON-LD 1.1 allows term definitions to be <em>protected</em>.
     </p>
   <p>A <dfn>protected term definition</dfn> is a term definition with a member <code>@protected</code> set to <code>true</code>.
-    It prevents further contexts to override this term definition.
-    </p>
+    It generally prevents further contexts to override this term definition,
+    either through a new definition of the same term,
+    or through clearing the context with <code>"@context": null</code>.
+    Such attempts will raise an error and abort the processing
+    (except in some specific situations described
+    <a href="#overriding-protected-terms">below</a>).
+  </p>
 
 
-  <aside class="example ds-selector-tabs changed"
-    title="A protected term definition can not be overridden">
-    <div class="selectors">
-        <button class="selected" data-selects="original">Original</button>
-        <button data-selects="expanded">Expanded</button>
-        <button data-selects="statements">Statements</button>
-        <button data-selects="turtle">Turtle</button>
-        <a class="playground" target="_blank"></a>
-    </div>
-    <pre class="original selected nohighlight" data-transform="updateExample">
-      <!--
+  <pre class="example" data-transform="updateExample"
```

Comment:
[According to ReSpec](https://github.com/w3c/respec/wiki/example) we can use `illegal-example` for examples of things that should not evaluate; perhaps this is a good place to use it.